UPGRADE-4.x.md

```diff
@@ -1,12 +1,19 @@
 UPGRADE 4.x
 ===========
 
-UPGRADE FROM 3.x to 3.x
+UPGRADE FROM 4.x to 4.x
 =======================
 
-### Support for NelmioApiDocBundle > 3.6 is added
+### Dependencies
 
-Controllers for NelmioApiDocBundle v2 were moved under `Sonata\UserBundle\Controller\Api\Legacy\` namespace and controllers for NelmioApiDocBundle v3 were added as replacement. If you extend them, you must ensure they are using the corresponding inheritance.
+- "sonata-project/datagrid-bundle" is bumped from ^2.4 to ^3.0.
+If you are extending these method you MUST add argument and return type declarations:
+  - `Sonata\UserBundle\Entity\UserManager::getPager()`
+  - `Sonata\UserBundle\Entity\GroupManager::getPager()`
+
+- "nelmio/api-doc-bundle" from now is also supported for ^3.6
```

Comment:
The term "now" is relative in a document like this. IMO, we should avoid to make use of these kind of references. The relevant concept here is the version number.
```suggestion
- Added support for "nelmio/api-doc-bundle" ^3.6
```